name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/solarisbank/SolarisbankAGPDFExtractorTest.java

```diff
@@ -34,46 +34,19 @@ public void testGiroKontoauszug01()
 
         assertThat(errors, empty());
```

Comment:
Add this
`
        assertThat(errors, empty());
        assertThat(countSecurities(results), is(0L));
        assertThat(countBuySell(results), is(0L));
        assertThat(countAccountTransactions(results), is(9L));
        assertThat(results.size(), is(4));
        new AssertImportActions().check(results, CurrencyUnit.EUR);
`